### ISSUE TITLE
issue on adding new pacakge  

### ISSUE THREAD

**Issue by YehualashetGit** [ever-co/ever-teams#1631]
Currently, we are encountering an issue when attempting to add  a new package into our apps. This issue results in the following example error message :

![image](https://github.com/ever-co/ever-teams/assets/25075676/d70a0ae1-4bf1-4363-97bb-b1991b224575)

**Comment by evereq** [comment]
@YehualashetGit what command do you run!?

**Comment by YehualashetGit** [comment]
> @YehualashetGit what command do you run!?

I have updated the issue with the command I run

**Comment by evereq** [comment]
@maceteligolden did you run ` yarn policies set-version 1.19.0` and result of PR https://github.com/ever-co/ever-teams/pull/1633 is what happens after you run that command? 

I found https://github.com/yarnpkg/yarn/issues/8405 and in one of comments they suggest to update nx / lerna packages to latest, did you checked that? https://github.com/yarnpkg/yarn/issues/8405#issuecomment-1341129303

**Comment by maceteligolden** [comment]
> @maceteligolden did you run ` yarn policies set-version 1.19.0` and result of PR #1633 is what happens after you run that command?
> 
> I found [yarnpkg/yarn#8405](https://github.com/yarnpkg/yarn/issues/8405) and in one of comments they suggest to update nx / lerna packages to latest, did you checked that? [yarnpkg/yarn#8405 (comment)](https://github.com/yarnpkg/yarn/issues/8405#issuecomment-1341129303)
> 
> I don't like idea to pin Yarn version that way, however seems others doing it too, e.g. https://github.com/vuetifyjs/vuetify/tree/master/.yarn/releases. Still if we can resolve this without adding yarn binary to our repo that would be preferred approach.
> 
> Also, maybe we just need to try yarn v2? :))) No idea if it will work well with all deps we have / with NX and with Lerna etc...

yes i updated the yarn policies. i'll check updating nx and lerna.
